registry.k8s.io/images/k8s-staging-kueue/images.yaml

```diff
@@ -1,11 +1,13 @@
 - name: charts/kueue
   dmap:
+    "sha256:02c4857f5c83cdb1274c8ab9f07f7c6f5e3eb6d2041228dbcfedd27ce76f863a": ["0.11.2"]
     "sha256:8c8406f4c10b606c09eaae02c3c5d79faf5f524fe58579ad0e403d125d67a491": ["0.11.1"]
     "sha256:d3107c3f20544cca627a7ed24597310c5464bf4acbfcc77d94dd9b546f07b038": ["0.11.0"]
     "sha256:c48a503b7411fb5f0dacc16938d34d442d2cf8299a81dbb6b7a205d53fdfb259": ["0.10.3"]
     "sha256:80591f127d58324a4a82967d3657237f1496f586d48580b74282bd83fa4747d3": ["0.9.5"]
 - name: kueue
   dmap:
+    "sha256:516dbe9dfba224936b1a9f01fd93f0d4a17de7284b4751803ac065a48b0d37c7": ["v0.11.2"]
```

Comment:
> docker run -it us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:v0.11.2
Unable to find image 'us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:v0.11.2' locally
v0.11.2: Pulling from k8s-staging-images/kueue/kueue
3d78e577de35: Already exists 
bfb59b82a9b6: Already exists 
0b1f95f08c81: Already exists 
a62778643d56: Already exists 
7c12895b777b: Already exists 
3214acf345c0: Already exists 
5664b15f108b: Already exists 
0bab15eea81d: Already exists 
4aa0ea1413d3: Already exists 
da7816fa955e: Already exists 
9aee425378d2: Already exists 
18185968b3e8: Pull complete 
Digest: sha256:516dbe9dfba224936b1a9f01fd93f0d4a17de7284b4751803ac065a48b0d37c7
Status: Downloaded newer image for us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:v0.11.2